src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -77,10 +77,16 @@ PROGRAM_INFO("AdaBoost",
     "predictions for a given test dataset.  A test dataset may be specified "
     "with the " + PRINT_PARAM_STRING("test") + " parameter.  The predicted "
     "classes for each point in the test dataset are output to the " +
-    PRINT_PARAM_STRING("output") + " output parameter.  The AdaBoost model "
-    "itself is output to the " + PRINT_PARAM_STRING("output_model") +
+    PRINT_PARAM_STRING("predictions") + " output parameter.  The AdaBoost "
+    "model itself is output to the " + PRINT_PARAM_STRING("output_model") +
     " output parameter."
     "\n\n"
+    "Note : The following parameter is deprecated and "
+    "will be removed in mlpack 4: " + PRINT_PARAM_STRING("output") +
+    "."  +
+    "\nUse " + PRINT_PARAM_STRING("predictions") + " instead of " +
+    PRINT_PARAM_STRING("output") + '.' +
+    "\n\n"
     "For example, to run AdaBoost on an input dataset " +
     PRINT_DATASET("data") + " with perceptrons as the weak learner type, "
     "storing the trained model in " + PRINT_MODEL("model") + ", one could "
```

Comment:
Below in the comment there's a use of `output`; can you change that to `predictions` too?

Reply:
Line No ?

Reply:
This line:

```
    PRINT_CALL("adaboost", "input_model", "model", "test", "test_data",
        "output", "predictions"),
```
:)

Reply:
Thanks :)